src/mcts/params.cc

```diff
@@ -538,10 +530,7 @@ void SearchParams::Populate(OptionsParser* options) {
       -0.6521f;
   options->Add<BoolOption>(kDisplayCacheUsageId) = false;
   options->Add<IntOption>(kMaxConcurrentSearchersId, 0, 128) = 1;
-  options->Add<IntOption>(kDrawScoreSidetomoveId, -100, 100) = 0;
-  options->Add<IntOption>(kDrawScoreOpponentId, -100, 100) = 0;
-  options->Add<IntOption>(kDrawScoreWhiteId, -100, 100) = 0;
-  options->Add<IntOption>(kDrawScoreBlackId, -100, 100) = 0;
+  options->Add<IntOption>(kDrawScoreId, -100, 100) = 0;
```

Comment:
Since we are overhauling this, why not make it a float option?

Reply:
Fully agree, and already considered that. Main question is whether the -100...100 still makes sense then, or whether we should switch to -1.0 ... 1.0

Reply:
Only [-1.0, 1.0] makes sense as a range to me, I hadn't considered anything else. I went back to find the justification for going to integer values, and it was that it looked better in guis, which I suppose is reasonable for 4 values that need to be set together.